x-pack/plugins/ml/public/application/components/node_available_warning/hooks.ts

```diff
@@ -25,7 +25,7 @@ export function useMlNodeAvailableCheck() {
 
 export function useMlNodeCheck() {
   const { http } = useKibana().services;
-  const ml = useMemo(() => mlApiServicesProvider(new HttpService(http!)), [http]);
+  const mlApi = useMemo(() => mlApiProvider(new HttpService(http!)), [http]);
```

Comment:
Is there a reason for this specific code or could this be `const mlApi = useMlApi()`?

Reply:
Yes, this is intentional, this component is shared out from the ML plugin and used in other plugins to warning about ML node availability.